R/html.R

```diff
@@ -563,7 +563,7 @@ parse_fig_labels = function(content, global = FALSE) {
   m = gregexpr(sprintf('\\(#((%s):[-/[:alnum:]]+)\\)', reg_label_types), content)
   labs = regmatches(content, m)
   cntr = new_counters(label_types, chaps)  # chapter counters
-  figs = grep('^<div class="figure', content)
+  figs = grep('^<div .*class="figure', content)
```

Comment:
Does it make sense to make the regex stricter?

```suggestion
  figs = grep('^<div [^>]*?class="figure', content)
```